DfE.FindInformationAcademiesTrusts/Pages/Cookies.cshtml.cs

```diff
@@ -63,12 +63,14 @@ private void ApplyCookieConsent()
 
         if (Consent is false)
         {
-            var optionalCookieMatches = new[] { "ai_session", "ai_user" };
-            foreach (var cookie in optionalCookieMatches.Where(cookie =>
-                         _httpContextAccessor.HttpContext!.Request.Cookies.ContainsKey(cookie)))
-            {
-                _httpContextAccessor.HttpContext!.Response.Cookies.Delete(cookie);
-            }
+            var optionalCookieMatches = new[] { "ai_session", "ai_user", "_ga", "_gid" };
```

Comment:
Very minor but could we rename this `optionalCookiePrefixes` since we've changed that matching behaviour?